cmd/frontend/graphqlbackend/webhooks.go

```diff
@@ -74,6 +75,43 @@ func (r *schemaResolver) Webhooks(ctx context.Context, args *struct {
 	return &webhookConnectionResolver{}
 }
 
+func (r *schemaResolver) Webhook(ctx context.Context, args *struct {
+	ID   *graphql.ID
+	UUID *string
+}) (*webhookResolver, error) {
+	if err := auth.CheckCurrentUserIsSiteAdmin(ctx, r.db); err != nil {
+		return nil, errors.New("must be site admin")
+	}
+
+	if (args.ID == nil && args.UUID == nil) || (args.ID != nil && args.UUID != nil) {
+		return nil, errors.New("either 1 of ID or UUID must be provided, but not both")
```

Comment:
Probably? (probably not)

```suggestion
		return nil, errors.New("either one of ID or UUID must be provided, but not both")
```